src/doc/trpl/documentation.md

```diff
@@ -15,7 +15,7 @@ comments":
 // the "link" crate attribute is currently required for rustdoc, but normally
 // isn't needed.
 #![crate_id = "universe"]
-#![crate_type="lib"]
+#![crate_type = "lib"]
```

Comment:
it's all good! no PR is too small. That guideline is WIP, and I would say goes _against_ most style stuff, so let's just leave this for now, since it's already here.